packages/react/src/components/planning-grid/PlanningGrid.tsx

```diff
@@ -78,7 +88,7 @@ export const PlanningGrid = React.forwardRef<DataGrid, PlanningProps>(({ dataSou
       <Column dataField='owner' caption='Owner' hidingPriority={5}>
         <RequiredRule />
       </Column>
-      <Column dataField='status' caption='Status' minWidth={120} cellRender={StatusTask} editCellRender={EditStatus} hidingPriority={3}>
+      <Column dataField='status' caption='Status' minWidth={120} cellRender={ItemField} editCellRender={editStatusRender} hidingPriority={3}>
```

Comment:
Since ItemField is a component, it's better to use the 'cellComponent' prop here
```
<Column dataField='status' caption='Status' minWidth={120} cellComponent={ItemField} editCellRender={editStatusRender}  hidingPriority={3}>
```